### ISSUE TITLE
Problem with Protect from DPI

### ISSUE THREAD

**Issue by Batman2814** [AdguardTeam/AdguardForAndroid#4221]
When having Protect from DPI setup with Adguard for Android MyBell Application through the google play store says No Network when trying to go into the app but if I press retry again for about more then 5 times it does bring me into My Bell App but shows could not load server so I have to press retry which it does load my account information. I went through all Adgaurd for Android settings to make sure it was Protect from DPI and it was it. when I have Protect from DPI disabled in Adguard for Android my My Bell app worked fine didn't give me those error messages.

**Comment by maxikuzmin** [comment]
@Batman2814 To troubleshoot this issue, we need to get additional logs.

Here's what we need you to do:

- Collect the debug log as it's [explained here](https://kb.adguard.com/en/android/solving-problems/log#debug);
- Remember the exact time when the issue was reproduced. We will need it to find the corresponding records in the log file;
- Additionally, capture the system bug report after reproducing the issue. [Here's the instruction](https://developer.android.com/studio/debug/bug-report);
- Send both files to devteam@adguard.com. Mention the Github issue number and the exact time when the issue was reproduced.

**Comment by Batman2814** [comment]
logs have been sent to email

**Comment by Batman2814** [comment]
logs have been sent to email

**Comment by maxikuzmin** [comment]
@Batman2814 we're looking into the problem. Please wait

**Comment by maxikuzmin** [comment]
@Batman2814 we're still looking into the problem. Please wait

**Comment by maxikuzmin** [comment]
@Batman2814 please try updating to version 3.6.10 and repeat the problem. Is it still repeating with the new version?

**Comment by Batman2814** [comment]
Its still repeating with version 3.6.10

**Comment by Batman2814** [comment]
any news on when this will be fixed?

**Comment by Batman2814** [comment]
Yes it does work. So can that be added in Adgaurd without putting it in the user rule?
